docs/getting_started/features/pathways.md

```diff
@@ -5,47 +5,212 @@ Pathways contains features that allow to model large transit stations, helping r
 
 Location types are used to classify key areas within transit stations such as exits/entrances, nodes or boarding areas, as well as their relationship. Location types serve as a fundation for modeling transit stations using Pathways.
 
+**Pre-requirements**: Implement base features.
+
 | Files included                   | Fields included   |
 |----------------------------------|-------------------|
-|[stops.txt](/schedule/reference/#stopstxt)|`location_type`, `parent_station` |
+|[stops.txt](/documentation/schedule/reference/#stopstxt)|`location_type`, `parent_station` |
+
+??? note "Sample Data"
+    <p style="font-size:16px">
+    The following table defines multiple nodes within a parent station in `stops.txt`, specifying platforms, entrances and exists and generic nodes.
```

Comment:
It would be nice to repeat "location types" in this definition. 

```suggestion
    The following sample defines multiple location types within a transit station in `stops.txt`: the parent station, representing the main location, and its child locations such as platforms, entrances/exists, and generic nodes.
```